rslib/backend.proto

```diff
@@ -66,6 +66,10 @@ message DeckConfigID {
     int64 dcid = 1;
 }
 
+message TagID {
+    int64 tid = 1;
+}
+
```

Comment:
Perhaps can be removed?

Reply:
I was going to go ahead and remove tag IDs. I initially thought this will be useful for some reason.

Reply:
Please hold off on that for now, I'll give it some thought. If they don't end up getting dropped, we'll need to remember to add an index on the name column, as currently it'll be doing a table scan.